activesupport/test/log_subscriber_test.rb

```diff
@@ -92,10 +92,11 @@ def test_event_attributes
   end
 
   def test_does_not_send_the_event_if_it_doesnt_match_the_class
-    ActiveSupport::LogSubscriber.attach_to :my_log_subscriber, @log_subscriber
-    instrument "unknown_event.my_log_subscriber"
-    wait
-    # If we get here, it means that NoMethodError was not raised.
```

Comment:
basically the comment was implying the same that could be achieved with explicit `assert_nothing_raised`